### PR TITLE
Jan 2021 cloud updates

### PR DESCRIPTION
**Pull request by kelseiv**
- Update Cloud monthly updates doc
- Update write data procedures for Cloud

### CODE REVIEW

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 1]
```diff
@@ -11,7 +11,80 @@ aliases:
...
+    - InfluxDB v2 Listener
+    - NSD
+    - OPC-UA
+    - Windows Event Log
+
+### Visualization updates
+
+  - Add new [Band Plot visualization](/influxdb/v2.0/visualize-data/visualization-types/band/).
+  - Usability improvements to visualizations:
+    - Add properties for storing your tick generation selections, including a `generateAxisTicks` property to turn this feature on and off.
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:62]: @bpalani, perhaps we should drop line 62 for now? 
@noramullen1 is currently updating documentation related to "Ticks" in visualizations

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 2]
```diff
@@ -11,7 +11,80 @@ aliases:
   - /influxdb/cloud/reference/release-notes/influxdb-cloud
 ---
 
-InfluxDB Cloud updates occur frequently. Find a compilation of monthly updates below.
+InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:14]: With this change, we may want to change the title/name of the doc in the front matter:

```diff
- title: InfluxDB Cloud monthly updates
+ title: InfluxDB Cloud updates
...
-     name: Monthly updates
+     name: InfluxDB Cloud updates
```

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 3]
```diff
@@ -11,7 +11,80 @@ aliases:
...
+- [CLI updates](#cli-updates)
+- [API updates](#api-updates)
+- [Task updates](#task-updates)
+
+### AWS and Microsoft regions
+
+- Add support for the following new regions:
+
+  - [AWS US East (Virginia) region](/influxdb/cloud/reference/regions/#amazon-web-services-aws)
+  - [Microsoft Azure regions](/influxdb/cloud/reference/regions/#microsoft-azure)
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:33]:
```suggestion
- Add support for Microsoft Azure and new AWS regions:
  - [Microsoft Azure](/influxdb/cloud/reference/regions/#microsoft-azure):
    - West Europe (Amsterdam) region
    - East US (Virginia) region
  - [AWS](/influxdb/cloud/reference/regions/#amazon-web-services-aws):
    - US East (Virginia) region
```

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 4]
```diff
@@ -11,7 +11,80 @@ aliases:
...
+
+### Flux updates
+
+- Update to [Flux v0.100.0](/influxdb/cloud/reference/release-notes/flux/#v0-100-0-2021-01-07).
+
+### InfluxDB community templates
+
+- Access any [InfluxDB community template](https://github.com/influxdata/community-templates#templates) directly in the Cloud user interface (UI). For more details, see how to [install and customize a template in the UI](/influxdb/cloud/influxdb-templates/cloud/).
+
+- Use the new [InfluxDB 2 Operational Monitoring community template](https://github.com/influxdata/community-templates/tree/master/influxdb2_operational_monitoring) to monitor InfluxDB 2.0.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:47]:
```suggestion
- Use the new [InfluxDB 2 Operational Monitoring community template](https://github.com/influxdata/community-templates/tree/master/influxdb2_operational_monitoring) to monitor InfluxDB OSS 2.0.
```

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md [hunk 5]
```diff
@@ -11,7 +11,80 @@ aliases:
...
+### CLI updates
+
+- Usability improvements to `influx` CLI:
+  - Add option to print raw query results in [`influx query`](/influxdb/cloud/reference/cli/influx/query/).
+  - Add ability to export resources by name using [`influx export`](/influxdb/cloud/reference/cli/influx/export/).
+  - Add new processing options and enhancements to [`influx write`](/influxdb/cloud/reference/cli/influx/write/).
+  - Add `--active-config` flag to [`influx`](/influxdb/cloud/reference/cli/influx/) commands to set the configuration for a single command.
+  - Add `max-line-length` flag to the [`influx write`](/influxdb/cloud/reference/cli/influx/write/) command to address "token too long" errors for large inputs.
+  - Add `--force` flag to the [`influx stacks rm`](/influxdb/cloud/reference/cli/influx/stacks/remove/) command, which lets you remove a stack without the confirmation prompt.
+  - Add `influxd` configuration options for storage options and InfluxQL coordinator tuning.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/cloud-updates.md:74]: Remove this line. Anything related to `influxd` doesn't apply to Cloud.

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud/reference/release-notes/cloud-updates.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/cloud/reference/release-notes/cloud-updates.md:14]: makes sense

**Review by bpalani — APPROVED** [review]

**Pull request merged**